Open_Source/6.Popular_Projects/6.18_Tensorflow.md

```diff
@@ -0,0 +1,78 @@
jAD/u4T/y6H/jgD/kQBATmX/gQA5SWD/dgCQl6IyQ1wtP1n/7d//mACboavAw8qFjZlHVWt+hpP/nQDIy9D/4sj/fQAoO1b/ngD/dwDj5Of/mQD/aQCtsrrs7e//1Lz/2KmkqbLZ29+2usF0fYxaZnjb3eH19vf/9u1ud4ceNFFRXnL/17b/wpP/0aj/5tT/lTr/mSb/p2X/qlP/vn3/jDP/nh3/n1//rUz/tYn/y67/gyv/sEL/nE3/zZj/u2z/YwD/3rj/p3X/xYn/6M//pTn/x5T/sGcRLExibX5lSnP+AAAHt0lEQVR4nO2bC1vbNhSGFWAjxolELg5uTAJ251wIkHVs69pt7Mrabs3//zs7R5IvIU6T7FlmM763T1tbsSzpjXQk2SAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADgSXP9pXhxWnYlngiv+p+Jz+kv2MiXByeHJOvg4OirsqtSde6+7h8eaFlHRwffvCi7OlXm+tvByeFBIuv4+OA1Qtc6Xg36J4d5WfXjo+/KrlQ1ue8PTk4eyarXj+sIXSvcvWn0+wWyiLcIXUtcf3856BfLarVa9R8QujJ+vKRutV4W6ULostw3GoPBJ2VdXLRav5ZdzSpw99Mludok6+Ksdf5QdlXLhoJVo7GVrLOzi5+vy65uubz8vbGtrPPziy/Krm65vLzcQdYZZEHWtuRlNX7pr8p6fVSHLEsqa9B4c3fdWJF19P70wzFkGRJZjcG9EKcFsmir83BRhyzGyGpc/sgna2QJ8Wu9BVlW1uX3ZgG1Vpa4/q0OWSzr8qc7e1Ik633y2c8tyPr98j49KZBVf52u2h/OW89c1v0fuZMiWfWjd+nn77CZziiU1aq38Ky0gGJZrVb97fvNmZ8b62S1WmdlV616rJf1tuyqVQ/I2oH1ss7Lrlr1gKwdKJRVh6xCCmQdP/x2DFlFFG+k33+oQ9Yq6546PFxclF216rH+Ec27DTmfIetlgRUgawcgawcgawcgawcgawcgawcgawcgawcgaxd+GTySdfDhmf+w36d48aafk3VUf/Y/RvppPj85SX/R6c+yK1N5rj/rn5Csw8O/MAK34PTr78Srb+42XwiYU4HfPwEAAAAAAAAAAAAAoGT85irjfRQ0elRIKETY6/b2Uta+GAfuY1RzHwV1lguSPSG60pXtfZS1L8bSMdQIc+TuR5ZMC9CFsCy35j4tWa6nqbEtc+jsT5aX4nSfoKyEJlW8u88CWJZaTnqysrjivX0W8D+W5Y/D8Sg5mUVRNOPEMOwMs2tmfJE/y98l6uQvsflGYTjcQla+SMoYZYezLHGptNJYktVWkiYqpWJz2vmobnsiVooSgxtb31kv4Itk0EuaJfx5IGmeU14nzRdMxFTJoL1J1qxr72bi5U2gbu1NqGx7jQzUx6t/udn/jJysqCYp2nPAl1OdQO10exOpk2qup9OuApeOXUpyA9sfuoG+gLIG5k4+ZbmZ8nXxBlkjvhvf33Fddj+WSQTle8z10SygG+3RwA5ksmZsQLqe4lbecAq3k3sM/WGD3N9myqk5atJdKEoJdN9q8lWUzPmkvhU1lBcLtHLbIOvqlu8rPckZJHXdKKg55kuhicfe35d7noK2J5M1YUk+HYQkQvGBnvVVTI3w+cvnL7rt0v88IK9YEY+dEX/xE2pW1KOkgAcRy6rR8rPd7HxaFvUpx+GSxiTbXdCB59QCPd65x0m9zKciZec/cbGRVNaQGh2Y0BCTEe5a3E5bT24/t2Lq1KSvU2gLIHl0UIrph1q3U7MXJ2mme7YTmpHIZI3Zo4l8Ppc+0h1K3/+Ke66uBBcQVCO+Z7KauUhPNQ2EbqcdFDT8qF00t81TWVEchqF1bEM9Rxc1MvEmjTK6e6a7nduhyGSx52RWnJj1Ho857q/8fZlKiCAJXuWTyvJSDboV6mpJlgiMrB41whvl8od0zTQ54RbH+eAsrKyUICdrZu+p0b1Rl6NvN3fc0HSykazOoiyVxaGnHcZMaPtPgawhhWRHeU0/WTdwh0y3SbG5mZ/LZ7c7iSyZk8VDLY1mkTLjfKEHHQV6FfVc7mux1L21EiSyuLLpaHFMrCqQxQ8ReKKXwTTU6T3TmQxjqePMiqziAK8nzTQxMPHL9Cfa5nucU+re+ih7eRTKIoI1smjSozUqT47S5S+cR16Y3IxlLXaSVUsTrayhYvk3Dl1Aw5T6lMzmitLJy8omLWK0RhbFmk6XFmM0699ePdq6ULdwJ1vLGqncanOWTBQu+9aaaEi6bapX9mWUTT5mqeHyZ2tkMVFMq30Oxe38dskGsC1lRUEtWxTwui2wH0qeE4V+5janw8fVKo9UFgX1LPgYVmUNfd+3VfdN65JZTEPTPce6LWUJXsYlE3Cse5QtlOZCXrOTTZcCviOqQior5t1K8j2bJwirssa3MljYFDt90dpb2gfqycp1W1ndRBDBc4oebtzfklUMLWGcyux1RE7WjLbDztysCMJAT/GrsiK77BTGDE1WPA5rUicNeb/UFdvL4gWtNOsO3mvZq+a8LTfHsd70VGSvI/J7w7HiKa47Hrfn1FiubfGitCbbo+GIY5ZZYPFjfEXZuslueGtZeg8up2En9tiV7Z9sn6cJoafGWmX2OmLpEU0z0CsoXhi4vIAvDPBzborUqwf70GYodRL/65jN5dayxEQL1kuR9N0ST5JJb/Kc3P6gfLrKlcls1qGtsX4DoxZ6OPJbrGQhdGv3daKnXPOaRk3sR9FC2WxTE/x9ypfJopPb5TJ70lV2udEOTFYpsxeJXJTtTW2VXlkFxua9Z3LWm3vetGm3F/x+NKkqvx81AW3YXniet2jnHl6OmlPPm3eTme0ql8+8ZF0tMwlEUUx3m9+EubEWZ7mHdGVlFg4AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADwX/gazIc4dkxJGbwAAAABJRU5ErkJggg==" alt="Sublime's custom image"/>
```

Comment:
Omg 😱, @Adi7171  Remove this Image link from `doc` or use a URL shortener, We don't need our whole covered with a single link

Reply:
## Also image not completely fetched :arrow_down: 

![Screenshot from 2021-10-04 21-23-17](https://user-images.githubusercontent.com/56113566/135883550-992a0fe6-7916-437a-852d-c3a401a61803.png)

Reply:
done